environment.yml

```
@@ -0,0 +1,11 @@
+name: fundingpips-scalper
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - pip
+  - pip:
+      - pytest
+      - ruff
+      - black
+      - coverage
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)